src/ui/Tab/Tab.ts

```diff
@@ -119,7 +120,16 @@ export class Tab extends Component {
      * If cached results are available but are older than the specified age, a new query will be performed on the index.<br/>
      * By default, this is left undefined and the Coveo Search API will decide the cache duration.
      */
-    maximumAge: ComponentOptions.buildNumberOption()
+    maximumAge: ComponentOptions.buildNumberOption(),
+    /**
+     * Specifies if the responsive mode should be enabled for the tabs. Responsive mode will make the overflowing tabs dissapear and instead
+     * be availaible using a dropdown button. Responsive tabs are enabled when tabs overflow or when the width of the search interface
+     * becomes too small. Disabling reponsive mode for one tab will disable it for all tabs.
+     * Therefore, this options only needs to be set on one tab to be effective.
```

Comment:
I would add 2 new line between paragraph to make it easier to read in the generated doc.